ui/components/app/edit-gas-fee-button/edit-gas-fee-button.js

```diff
@@ -80,23 +80,22 @@ export default function EditGasFeeButton({ userAcknowledgedGasMissing }) {
         <InfoTooltip
           contentText={
             <div className="edit-gas-fee-button__tooltip">
-              <Typography
-                variant={TYPOGRAPHY.H7}
-                color={COLORS.TEXT_ALTERNATIVE}
-              >
-                {t('dappSuggestedTooltip', [transaction.origin])}
-              </Typography>
+              {transaction?.origin && (
+                <Typography
+                  variant={TYPOGRAPHY.H7}
+                  color={COLORS.TEXT_ALTERNATIVE}
+                >
+                  {t('dappSuggestedTooltip', [transaction.origin])}
+                </Typography>
+              )}
               <Typography variant={TYPOGRAPHY.H7}>
-                <b>{t('maxBaseFee')}</b>
-                {maxFeePerGas}
+                <b>{t('maxBaseFee')}</b> {maxFeePerGas}
```

Comment:
In the future we should move to `<strong>` but that's an existing issue so not going to block.

Reply:
Interesting and noted. I would love to hear more about this sometime since [it seems to be a controversial decision](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em).